app/Http/Controllers/ReportController.php

```diff
@@ -78,7 +78,7 @@ public function index(Request $request)
         if ($request->ajax()) {
             return Datatables::of((new ReportDataTable())->get($request->only(['filter_created_by'])))->make(true);
         }
-        $users = User::orderBy('name')->pluck('name', 'id');
+        $users = User::whereIsEmailVerified(true)->orderBy('name')->pluck('name', 'id');
```

Comment:
```suggestion
        $users = User::whereIsActive(true)->whereIsEmailVerified(true)->orderBy('name')->pluck('name', 'id');
```

Reply:
also, check here permissions as well. if the user hasn't report permission than this query should not fire.

```
$users = [];
  if(Auth::user()->hasPermissionTo('manage_reports')) {
             $users = User::whereIsActive(true)->whereIsEmailVerified(true)->orderBy('name')->pluck('name', 'id');
    }
```